opm/autodiff/FlowMainEbos.hpp

```diff
@@ -56,13 +56,30 @@
 #include <dune/common/parallel/mpihelper.hh>
 #endif
 
+BEGIN_PROPERTIES;
```

Comment:
`;` caused some warnings `FlowMainEbos.hpp:73:15: warning: extra ‘;’ [-Wpedantic]`. 

There are several places of this thing.

Reply:
I don't get these warnings for some reason. would be nice if you made a patch.